ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```diff
@@ -940,6 +940,12 @@ keys:
                   type: str
                 activate:
                   type: bool
+                rcf_in:
+                  type: str
+                  description: Inbound rcf function name without parenthesis.
```

Comment:
Since EOS supports arguments in some cases, we should let the str here be _with_ parentheses, so the user could also do `MyFunction(myarg)`

Reply:
same on all the other RCF PRs

Reply:
Made the changes on this PR

Reply:
Made changes in all the RCF PRs :)